### PR TITLE
Some last minute ISA updates for Cerebellin

### PR DESCRIPTION
**Pull request by djcomlab**
Some last minute ISA updates for Cerebellin
- fixes container version for ISA create tool
- fixes to the ISA create tool XML

### CODE REVIEW

**Comment by pcm32** [comment]
Is this ready for review? Does it run well in minikube? Thanks!

**Comment by djcomlab** [comment]
Yep, tested in minikube and seems to run fine.

File: tools/phenomenal/isatools/isa_create_metabo/isa_create_metabo.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
     <requirements>
```
Comment (pcm32) [inline tools/phenomenal/isatools/isa_create_metabo/isa_create_metabo.xml]: I think that having these requirements is what confuses the container selector when using dynamic destinations...

**Comment by pcm32** [comment]
@djcomlab please delete your local branch named isa-updates-for-cerebellin. Thanks.

**Pull request merged**